src/chains/migrations/0038_auto_20221024_1543.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.0.5 on 2022-10-24 15:43
+
+from django.apps.registry import Apps
+from django.contrib.auth.models import Group, Permission
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+
+def create_support_group(apps: Apps, schema_editor: BaseDatabaseSchemaEditor) -> None:
+    support_group, created = Group.objects.get_or_create(name="support")
+    permission = Permission.objects.filter(codename="change_chain")
+    if permission:
```

Comment:
Permission must exist we shouldn't check this

Reply:
I was getting an error when running tests due to this, that's why I included the condition. Maybe it's related with not having the permission defined in `Meta` as you said in [the previous comment](https://github.com/safe-global/safe-config-service/pull/688#discussion_r1004205828)...

Reply:
That's it. Permission should be created in a migration when adding it to `Meta`, and you can use that migration to add it to the group. Also, if possible, do everything in the same migration

Reply:
@Uxio0 I was testing a lot trying to understand why the migration was failing (because the permission wasn't found), I think the reason why we need `Permission.objects.get_or_create` here instead of simply `Permission.objects.get` is because the permissions data is not inferred/written to the database until all migrations are executed (using the [post_migrate](https://docs.djangoproject.com/en/4.1/ref/signals/#post-migrate) signal if I'm not wrong.

Reply:
I think [this issue](https://code.djangoproject.com/ticket/23422) is related. This also explains why test were failing in my local setup (I understand the whole list of migrations gets executed against a test database when running the tests).